mscore/shortcut.cpp

```diff
@@ -2499,8 +2499,8 @@ Shortcut Shortcut::_sc[] = {
          MsWidget::SCORE_TAB,
          STATE_NORMAL | STATE_NOTE_ENTRY_STAFF_PITCHED | STATE_NOTE_ENTRY_STAFF_DRUM /*| STATE_NOTE_ENTRY_TAB*/,
          "dec-duration-dotted",
-         QT_TRANSLATE_NOOP("action","Decrease Duration to Next Smaller Dotted or Non-Dotted Duration"),
-         QT_TRANSLATE_NOOP("action","Decrease duration to next smaller dotted or non-dotted duration"),
+         QT_TRANSLATE_NOOP("action","Decrease Duration Dotted"),
+         QT_TRANSLATE_NOOP("action","Decrease duration dotted"),
```

Comment:
The command's description has been simplified, for reasons I mentioned in: https://musescore.org/en/node/175461#comment-682611